### ISSUE TITLE
Bash/WSL 80070005 - unprivileged user only

### ISSUE THREAD

**Issue by nickofferman** [Microsoft/WSL#3775]
Windows build number: Version 10.0.17134.523

 Running WSL either by Start->Run->wsl->enter or opening cmd.exe and typing wsl or bash and hitting enter.  All result in 
```
Error: 0x80070005
Press any key to continue...
```

Unless I launch them as Administrator, in which case I get a bash prompt, but as root.

I have uninstalled Ubuntu1804, uninstalled WSL, rebooted, reinstalled WSL, rebooted, reinstalled Ubuntu 1804, and rebooted.

I am doing this through Chocolatey and we have the Windows Store removed in our environment.  I just recently wiped the machine and started clean.  I cannot remember whether I had this running after the reinstall and it now does not work after the weekend(presumably updates?) or if I did not ever confirm it was working in the new install.

I did try stopping Symantec and then launching bash, but that did not help with the issue.
Please see attached files captured with the start_lxcore_trace script. 
[lxcore-trace-files.zip](https://github.com/Microsoft/WSL/files/2759977/lxcore-trace-files.zip)

Any suggestions are appreciated.

**Comment by benhillis** [comment]
This is interesting, it looks like setting some extended attributes on your rootfs folder is failing with access denied. I'm not sure how things got into this state, you could probably recover this by making sure that the rootfs folder for 1804 has write access for your non-admin user.

**Comment by nickofferman** [comment]
Thanks!  After looking around at the rootfs(finding it was a trick).  I added myself manually to no avail.

Eventually, I made my way to the [Chocolatey package page](https://github.com/bcurran3/ChocolateyPackages/blob/master/wsl-ubuntu-1804/readme.md) and looked at the comments. This led me to the packages notes(I wasn't aware there are such things).  The installer uses --root and does not create a normal user.  They tell note that it is required for the end user to manually create a user.
Mystery solved.

I removed Ubuntu from the machine, download the AppX package, and installed via PowerShell.  Works better/as expected.

Thanks for the help.
